docs/.vuepress/config/locales/index.js

```diff
@@ -1,5 +1,5 @@
-const langs = ['en', 'zh', 'tr'];
-const paths = { en: '/' };
+const langs = ['en', 'zh', 'tr','it'];
+const paths = { it: '/it/' };
```

Comment:
This should be restored to how it was before

Reply:
In fact, the path does not need to be modified

Reply:
`langs` should stay as it is now (with the `it` added), but `paths` should be changed back; this configures it so that the english docs will show up at `docs.flarum.org` instead of `docs.flarum.org/en` since those are the default

Reply:
oh, sorry for that, done!